docs/docs/getting-started/first-steps.md

```diff
@@ -40,45 +40,61 @@ If you don't have a cloud subscription, check out [MiniConstellation](first-step
+    <tabItem value="azure-command" label="Azure">
+
+    Create the IAM configuration on Azure with the following command (You must be authenticated with the [Azure CLI](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) in the shell session):
 
     ```bash
-    RESOURCE_GROUP=constellation # enter name of new resource group for your cluster here
-    LOCATION=westus # enter location of resources here
-    SUBSCRIPTION_ID=$(az account show --query id --out tsv)
-    SERVICE_PRINCIPAL_NAME=constell
-    az group create --name "${RESOURCE_GROUP}" --location "${LOCATION}"
-    az group create --name "${RESOURCE_GROUP}-identity" --location "${LOCATION}"
-    az ad sp create-for-rbac -n "${SERVICE_PRINCIPAL_NAME}" --role Owner --scopes "/subscriptions/${SUBSCRIPTION_ID}/resourceGroups/${RESOURCE_GROUP}" | tee azureServiceAccountKey.json
-    az identity create -g "${RESOURCE_GROUP}-identity" -n "${SERVICE_PRINCIPAL_NAME}"
-    identityID=$(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query principalId --out tsv)
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Virtual Machine Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    az role assignment create --assignee-principal-type ServicePrincipal --assignee-object-id "${identityID}" --role 'Application Insights Component Contributor' --scope "/subscriptions/${SUBSCRIPTION_ID}"
-    echo "subscription: ${SUBSCRIPTION_ID}
-    tenant: $(az account show --query tenantId -o tsv)
-    location: ${LOCATION}
-    resourceGroup: ${RESOURCE_GROUP}
-    userAssignedIdentity: $(az identity show -n "${SERVICE_PRINCIPAL_NAME}" -g "${RESOURCE_GROUP}-identity" --query id --out tsv)
-    appClientID: $(jq -r '.appId' azureServiceAccountKey.json)
-    clientSecretValue: $(jq -r '.password' azureServiceAccountKey.json)"
+    constellation iam create azure --region=westus --resourceGroup=constellTest --servicePrincipal=spTest
     ```
 
-    Fill the values produced by the script into your configuration file.
+    Find an exact command description in our [CLI reference](https://docs.edgeless.systems/constellation/reference/cli).
 
-    :::tip
+    By default, Constellation uses `Standard_DC4as_v5` CVMs (4 vCPUs, 16 GB RAM) to create your cluster. Optionally, you can switch to a different VM type by modifying **instanceType** in the configuration file. For CVMs, any VM type with a minimum of 4 vCPUs from the [DCasv5 & DCadsv5](https://docs.microsoft.com/en-us/azure/virtual-machines/dcasv5-dcadsv5-series) or [ECasv5 & ECadsv5](https://docs.microsoft.com/en-us/azure/virtual-machines/ecasv5-ecadsv5-series) families is supported.
 
-    Alternatively, you can leave `clientSecretValue` empty and provide the secret via the `CONSTELL_AZURE_CLIENT_SECRET_VALUE` environment variable.
+    You can also run `constellation config instance-types` to get the list of all supported options.
 
-    :::
```

Comment:
The IAM create command is not going to set this environment variable, should I rework the command to do that?

Reply:
I think it sounds like something which would be set automatically, if we say "**the** `CONSTELL_AZURE_CLIENT_SECRET_VALUE`". 
Also i think we should explain why an environment variable is preferred here.

I would phrase it like "Since this is a sensitive value, alternatively you can leave `clientSecretValue` empty and provide the secret via an environment variable such as `CONSTELL_AZURE_CLIENT_SECRET_VALUE`."

Probably you can improve that, but generally I would provide this information to the user, since i personally didn't get the grasp of it at first when reading it myself.

Reply:
Stating the reason makes sense!

"an environment variable such as" would be wrong because it's about exactly this variable:
https://github.com/edgelesssys/constellation/blob/be01cf7129d6f5ebcc87244aec8f3a94dc6c2d89/internal/constants/constants.go#L133-L135

Feel free to rephrase if you can make this more clear.

Reply:
Got it. Makes sense.

"Since this is a sensitive value, alternatively you can set the `CONSTELL_AZURE_CLIENT_SECRET_VALUE` environmental variable, leave `clientSecretValue` empty in the `constellation-conf.yml` and provide the secret via the created environment variable."

Does that sound good?

Reply:
IMO still sounds a bit confusing. Maybe this is clearer (though a bit verbose)?

"Since this is a sensitive value, alternatively you can leave `clientSecretValue` empty in the configuration file and pass it via an environment variable instead. To this end, create the environment variable `CONSTELL_AZURE_CLIENT_SECRET_VALUE` and set it to the secret value."

Reply:
Sounds good!